src/playground/sample-data-ingestion/generator.py

```diff
@@ -0,0 +1,187 @@
+from faker import Faker
+
+# Standard libraries
+import shutil
+import json
+import gzip
+import csv
+import ast
+import os
+
+# Function that generates fake values for given fields
+# This is based off of the Faker library 
+def chooseField(kind, args=None):
+    fake = Faker()
+    
+    # "data-types.json" defines what enumerated types return in Faker
+    # Need support for the following:
+    #   "null": "return None",
+    #   "keyword": "return chooseField",
+    with open("data-types.json", "r") as f:
```

Comment:
are you assuming that data-types.json file in the same level of py ? it is better to create a good folder structure to host all different type of files, like as json file should be in separate folder, py in separate folder, and test.py in different folder